packages/docs/src/helpers/CodeSandboxExample.ts

```diff
@@ -37,31 +41,30 @@ const getCodeSandboxHtml = ({ requireIcons = false }: CodesandboxConfig): string
     >
     ${requireIcons ? iconsStyles : ''}
     <div id="app"></div>
+    <script type="module" src="/src/main.js"></script>
   `
 }
 
 const getCodeSandboxMain = ({ requireIcons = false }: CodesandboxConfig): string => {
-  if (requireIcons) {
-    return iconsConfig
-  }
+  if (requireIcons) { return iconsConfig }
 
   return main
 }
 
 const packageJson = ({ dependencies = {}, devDependencies = {} }: CodesandboxConfig): string => {
   const commonDeps = {
-    'core-js': '^3.6.5',
-    vue: '^3.0.0',
+    vue: 'latest',
     'vuestic-ui': `${packageUi.version}`,
   }
   const commonDevDeps = {
-    '@vue/cli-plugin-babel': '~4.5.0',
-    '@vue/cli-service': '~4.5.0',
-    '@vue/compiler-sfc': '^3.0.0',
+    '@vitejs/plugin-vue': 'latest',
```

Comment:
Maybe cap to major version? `latest` seems too risky for me.

Reply:
~3.0.0 I think would be nice. It will be a place where we can ensure that build doesn't work with last vite version.